components/camel-zeebe/pom.xml

```diff
@@ -0,0 +1,158 @@
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.camel</groupId>
+        <artifactId>components</artifactId>
+        <version>3.21.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>camel-zeebe</artifactId>
+    <packaging>jar</packaging>
+    <name>Camel :: Zeebe</name>
+    <description>Camel Zeebe support</description>
+
+    <properties>
+        <firstVersion>3.20</firstVersion>
+        <supportLevel>Experimental</supportLevel>
+        <zeebe.version>8.1.0</zeebe.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-support</artifactId>
+        </dependency>
+
+        <!-- jackson -->
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
+            <version>${jackson2-version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <version>${jackson2-version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>${jackson2-version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>io.camunda</groupId>
+            <artifactId>zeebe-client-java</artifactId>
+            <version>${zeebe.version}</version>
+        </dependency>
+
+        <!-- test dependencies -->
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-slf4j-impl</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-junit-jupiter</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-test-junit5</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <excludes>
+                        <exclude>**/*IntegrationTest.java</exclude>
```

Comment:
Our standard is to name Integration tests as `SomeThingIT.java` (i.e.: **IT.java). Our default surefire configuration is set to skip that and the failsafe is configured to include that. This also impacts a bit the code analysis on Sonar. 

Please, can you rename the integration tests to match this, so we avoid this extra code?

Reply:
Obs.: it's component-specific configurations for surefire and failsafe are OK to keep if they are needed.

Reply:
I tried renaming the integration tests to IT but then I was not able to disable them through the profile and they failed when the service is not present, which I guess will be the case for almost everybody.
Is there a way to name them IT and insure that they are only executed when the zeebe-integration profile is active?

Reply:
You can use JUnit's `@DisabledIfSystemProperty` or `@EnabledIfSystemProperty`. 

Here's an example: https://github.com/apache/camel/blob/main/components/camel-kafka/src/test/java/org/apache/camel/component/kafka/integration/health/KafkaConsumerBadPortHealthCheckIT.java#L61-L62

Reply:
Ok, changed the behavior to use EnbaledIfSystemProperty. Works fine. Thank you